boltdesignsystem/bolt
# DS-299 new brand icons

## Jira

https://pegadigitalit.atlassian.net/browse/DS-299

## Summary

Adds a group of brand icons to the Icon component.

## Details

1. 22 new icons from branding added.
2. Certain old icons have been renamed and deprecated. The old names point to the new names in the twig template.

## How to test

Run the branch locally and go through the icon name variations doc and the new Icon Pega Branding page under Icon docs. Make sure all icons are rendering as expected.

## Release notes

The following icon names have been deprecated, please update to the new names:

1. `discussions` renamed to `pega-chat-typing`
2. `questions` renamed to `pega-chat-question`
3. `healthcare` renamed to `pega-healthcare`

### Visual changes

The following icons have slight visual updates:

1. `discussions`/`pega-chat-typing` stroke is now thinner to match new brand icons
2. `questions`/`pega-chat-question` stroke is now thinner to match new brand icons
3. `healthcare`/`pega-healthcare` is converted to the new brand icon
